packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -607,7 +607,7 @@ exports.examples = ([
     },
   },
   {
-    title: 'Colored highlight/cursor for text input',
+    title: 'Colored highlight for text input',
```

Comment:
On iOS, the title is still true. If we were to change it, I'd want a diff tag. I think it might be better to modify the example with some macOS specific note / textinput with default text that says that iOS and macOS differ. I'm not _too_ sure what would work best.

Reply:
The example now keeps the original iOS title, but adds a note warning about the different behavior on macOS.
<img width="1136" alt="Screenshot 2023-04-13 at 23 18 18" src="https://user-images.githubusercontent.com/151054/231885341-87e13726-7ad9-4211-ba71-d5f50c729970.png">